Sources/Widgets/Manipulators/AbstractManipulator/index.d.ts

```diff
@@ -0,0 +1,209 @@
+import { vtkObject } from "../../../interfaces";
+import { vtkOpenGLRenderWindow } from "../../../Rendering/OpenGL/RenderWindow"
+
+/**
+ *
+ */
+export interface IAbstractManipulatorInitialValues {
+    userOrigin?: number[];
+    handleOrigin?: number[];
+    widgetOrigin?: number[];
+    userNormal?: number[];
+    handleNormal?: number[];
+    widgetNormal?: number[];
+}
+
+export interface vtkAbstractManipulator extends vtkObject {
+
+    /**
+     * Get the normal of the line
+     */
+    getNormal(callData: any): number[];
+
+    /**
+     * Get the origin of the line
+     */
+    getOrigin(callData: any): number[];
```

Comment:
I would recommend using Vector3 from `interfaces.d.ts` for these arrays, since it'll always be in 3D.